charts/dftbackend/Chart.yaml

```diff
@@ -1,3 +1,23 @@
+#################################################################################
+# Copyright (c) 2021,2022 Catena-X
```

Comment:
Catena-x is not providng (c) directly. Its normally, as far as i know, the company you are working with

Reply:
Hi @Siegfriedk, sorry I have just reused the existing License Header of this file:
https://github.com/eclipse-tractusx/dft-backend/blob/main/charts/dftbackend/values.yaml
Could you please suggest how the header in this project should look like?

Then we might also change the values.yaml file.

In my project for example we have:

#  Copyright (c) 2022, 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
#  Copyright (c) 2022, 2023 ZF Friedrichshafen AG
#  Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation

Thanks in advance.

Reply:
You would need to check the file history or ask the original owner of this file. As far as i know it would be t-systems

Reply:
Hi @Siegfriedk, I have updated the Chart.yaml and values.yaml file to reflect our discussion. Please review again.